src/components/badge/badge.css

```diff
@@ -0,0 +1,57 @@
+@import url('/src/components/utils.css');
+@import url('/src/components/avatar/avatar.css');
+
+.badge-container{
+    position : relative ;
+}
+.badge-status {
+    position: absolute;
+    min-width: 1.8rem;
```

Comment:
Badge on avatars look very small, increase its size

Reply:
thanks for the suggestion